### ISSUE TITLE
Feedback/Tooltip/Events for links (@click)

### ISSUE THREAD

**Issue by PlayerG9** [Textualize/textual#3224]
I am working on an app where it would be very helpful to be able to show what links do and where they go.
For widgets I have already found that you can set `.tooltip`, but for links I have found nothing like that.
Would be nice if you can set tooltip for links. Or an event like `on_link_enter` is sent, where you can get more infos like the action.

(https://textual.textualize.io/guide/actions/#links)

**Comment by PlayerG9** [comment]
any updates yet or did I get ignored?

**Comment by davep** [comment]
I see no updates. That, however, doesn't imply your request that someone do this work has been ignored; it simply means that nobody has been in a position to implement such a feature.

**Comment by TomJGooding** [comment]
> any updates yet or did I get ignored?

I'm so fed up with seeing these sorts of comments on open source projects. 

Remind yourself that you get to use Textual for free.

**Comment by PlayerG9** [comment]
I agree with you and am happy to use this library. But considering that my ticket got no label or any kind of reply for half a year it seems appropriate to ask again for updates.

(And yes. "overlooked" may have been the better word instead of "ignored")

**Comment by willmcgugan** [comment]
A prompt after a while is fine. We can lose track of requests now and again.

**Comment by willmcgugan** [comment]
There is no `on_link_enter`, but you can handle any [mouse event](https://textual.textualize.io/api/events/#textual.events.MouseEvent). The event object has a `style` attribute which is the style under the cursor. The `style` attribute has a `meta` dict which contains information regarding links.

So you could get link under the cursor with something like this (untested):

```python
if event.style is not None:
    click_action = event.style.meta["@click"]
```
